### PR TITLE
New Isolations

### PR DESCRIPTION
**Pull request by bgmoshe**
Will change branch name. This is without getting out - only isolating first circle and implementing 2 types of Isolations: Hotel and Home

### CODE REVIEW

File: src/corona_hakab_model/consts.py [hunk 1]
```diff
@@ -219,8 +221,24 @@ class Consts(NamedTuple):
     ]
     should_isolate_positive_detected: bool = False
     isolate_after_num_day: int = 1  # will be in isolation the next day.
-    p_will_obey_isolation: float = 1.0  # 100% will obey the isolation.
-    isolation_factor: float = 0.0  # reduce agent's relations strength by a factor.
+    sick_to_p_obey_isolation: Dict[bool, float] = {
+        True: 1.0,  # 100% sick will obey the isolation.
+        False: .95  # If not sick, 95% to obey isolation
+    }
+    isolation_factor: IsolationFactorsType = {
```
Comment (meirhalachmi) [inline src/corona_hakab_model/consts.py:228]: re-write the comment "# reduce agent's relations strength by a factor."

File: src/corona_hakab_model/consts.py [hunk 2]
```diff
@@ -340,74 +358,86 @@ class ImmuneTerminalState(ImmuneState, TerminalState):
...
             detectable=True,
+            has_symptoms=True,
             contagiousness=self.improving_health_infection_ratio,
             test_willingness=self.improving_health_test_willingness
         )
 
         pre_recovered = ContagiousStochasticState(
             self.PRE_RECOVERED,
             detectable=True,
+            has_symptoms=True,
```
Comment (meirhalachmi) [inline src/corona_hakab_model/consts.py]: Not sure about that. Make sure with someone

File: src/corona_hakab_model/manager.py [hunk 3]
```diff
@@ -158,37 +166,116 @@ def step(self):
 
         self.simulation_progression.snapshot(self)
 
-    def progress_tests_and_isolation(self, new_tests: List[PendingTestResult]):
+    def progress_isolations(self):
```
Comment (meirhalachmi) [inline src/corona_hakab_model/manager.py:165]: Don't you think that at least some of this logic should be in the HealthCareManager?

**Review by meirhalachmi — CHANGES_REQUESTED** [review]
Didn't finish reviewing it yet

**Comment by bgmoshe** [comment]
@meirhalachmi  Oops ....

**Pull request closed without merging**